../../../pobj/libdex-0.9.1/libdex-0.9.1/src/meson.build:102:21: ERROR: Unknown variable "libucontext_dep".

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -56,6 +56,8 @@ if cc.has_header('ucontext.h')
     libucontext_dep = disabler()
   endif
   config_h.set('HAVE_UCONTEXT_H', 1)
+else
+  libucontext_dep = disabler()
 endif
 
 if host_machine.system() == 'darwin'
